admin/controllers/school.js

```diff
@@ -29,6 +29,35 @@ const getHome = async (req, res, next) => {
   })
 }
 
+const getPupils = async (req, res, next) => {
+  res.locals.pageTitle = 'Pupil register'
+  const { sortOrder } = req.params
+  res.locals.sortOrder = sortOrder || 1
```

Comment:
I think sortOrder needs more validation - what if someone passes in unusual input?